### PR TITLE
This PR has various small fixes

### PR DESCRIPTION
**Pull request by sleshchenko**
This PR has various small fixes which are located in their self-descriptive commits.
Please merge it with rebase to keep the history.

### CODE REVIEW

File: docs/modules/user-guide/partials/proc_migrating-schema-version.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ This section describes how to migrate the existing schema version to a v2.x devf
 
 .Procedure
 
-. To migrate a schema version from a v1.x devfile to a v2.x devfile, replace `apiVersion: 1.0.0` with `schemaVersion: 2.0.0`:
+. To migrate a schema version from a v1.x devfile to a v2.x devfile, replace `apiVersion: 1.0.0` with `schemaVersion: 2.1.0`:
```
Comment (johnmcollier) [inline docs/modules/user-guide/partials/proc_migrating-schema-version.adoc:9]: Should we include information on migrating from v2.0 to v2.1 as well?

**Review by johnmcollier — COMMENTED** [review]
Generally looks good, just left one small comment.

File: docs/modules/user-guide/partials/proc_migrating-schema-version.adoc (hunk 1 above)
Comment (sleshchenko) [reply docs/modules/user-guide/partials/proc_migrating-schema-version.adoc:9]: That's a good catch I'm not sure about.
I expect that 2.1 comparing to 2.0 is an extension.
Users can just update schemaVersion to 2.1.0 and use newer format features.

**Review by jc-berger — APPROVED** [review]
LGTM from docs point of view. Good work!

File: docs/modules/user-guide/partials/proc_adding-commands-to-a-devfile.adoc [hunk 2]
```diff
@@ -88,7 +88,7 @@ projects:
 components:
   - name: maven
     container:
-      image: eclipe/maven-jdk8:latest
+      image: eclipse/maven-jdk8:latest
```
Comment (amisevsk) [inline docs/modules/user-guide/partials/proc_adding-commands-to-a-devfile.adoc:91]: We should consider including a registry here for clarity

**Review by amisevsk — APPROVED** [review]
LGTM, thanks for the separate commits :+1:

File: docs/modules/user-guide/partials/proc_adding-commands-to-a-devfile.adoc (hunk 2 above)
Comment (sleshchenko) [reply docs/modules/user-guide/partials/proc_adding-commands-to-a-devfile.adoc:91]: @amisevsk Do you mean to make it clear that it registry can be specified here as well, or that eclipse che images are on quay.io?

Comment (amisevsk) [reply docs/modules/user-guide/partials/proc_adding-commands-to-a-devfile.adoc:91]: Both I suppose. I think we're generally trying to move away from the implicit `docker.io` when there's no registry, but I don't actually know where `eclipse/maven-jdk8` is stored.

Comment (sleshchenko) [reply docs/modules/user-guide/partials/proc_adding-commands-to-a-devfile.adoc:91]: I was not able to find maven-jdk8, I think we can replace it with `quay.io/eclipse/che-java11-maven:nightly`

**Pull request merged**